pyproject.toml

```diff
@@ -16,6 +16,13 @@ langchain = ">=0.0.166"
 urllib3 = "<=1.26.15"  # added due to poetry errors
 python-dotenv = "^1.0.0"
 
+[tool.poetry.group.dev.dependencies]
+pre-commit = "^3.3.1"
+ruff = "^0.0.265"
+black = "^23.3.0"
+isort = "^5.12.0"
+prettier = "^0.0.7"
```

Comment:
i think we can remove these, right? 

they are part of the pre-commit config file. might be redundant to add here.

Reply:
yes sure, we can remove them, it's just a personal preference to see everything in the pyproject.

Reply:
@jpdus we can still keep `pre-commit` as dev dependency. others not needed imo.

i've already approved the PR. up to you if you want to add it back.